### ISSUE TITLE
Review documentation

### ISSUE THREAD

**Issue by maurolepore** [forestgeo/fgeo.abundance#73]
You can read the documentation of the package __fgeo.abundance__ [here](https://forestgeo.github.io/fgeo.abundance). To suggest edits, please add your comments below or create a  new [pull request](https://github.com/forestgeo/fgeo.abundance/pulls) ([see how](https://help.github.com/articles/about-pull-requests/)).

If you have any question, please ask me by referring to me as @maurolepore.

---

Search related issues on <https://github.com> with: `label:hacktoberfest maurolepore`

![image](https://user-images.githubusercontent.com/5856545/47038227-eb4ac400-d14e-11e8-916a-4f002dd507cd.png)

**Comment by ThomasSmithIRE** [comment]
@maurolepore, PR #77 created to fix 2 minor typos in readme.

**Comment by maurolepore** [comment]
@overstreeth, thank you very much for offering to review the documentation of packages that are approaching release. 

Could you please review each link under [Reference](https://forestgeo.github.io/fgeo.abundance/reference/index.html)?

**Comment by ToniaSeenauth** [comment]
Implement title case and update documentation title from "Calculate Abundance and basal area" to "Calculate Abundance and Basal Area".

**Comment by maurolepore** [comment]
Thank you @ToniaSeenauth! You make me realize the title of this package had a case inconsistent with other [__fgeo__ packages](https://forestgeo.github.io/fgeo/).

I have changed the title (see commit 779ad514914bc60fa072f08dfcd3850335077fc2) but in the opposite direction. While it is more appropriate to use Title Case, I prioritize that it is easier to maintain text in multiple places when the case is always lowercase (or as much lowercase as  I can get away with). I use this approach also when naming variables, and other things. This way I don't need to remember what I did here or there before deciding what case to use. Just one little system that helps me keep things simple.
